### PR TITLE
More fish install instructions for Linux, add Homebrew link

### PR DESCRIPTION
**Pull request by herrbischoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,15 +141,61 @@ end
 
 ### 2. How do I install fish on OS X?
 
-With Homebrew.
+With [Homebrew][Homebrew].
```
Comment (bucaran) [inline README.md]: Markdown tricks: Don't need to repeat [Homebrew].

File: README.md [hunk 2]
```diff
@@ -141,15 +141,61 @@ end
...
+pacman -S fish
+```
+
+#### Gentoo
+
+```sh
+emerge fish
+```
+
+#### From source
```
Comment (bucaran) [inline README.md]: Instead of "from source"

#### Autotools Build

**Comment by bucaran** [comment]
@herrbischoff Yay! This is great.

Would you mind adding these new sections to _all_ the other README's? 😅

**Comment by bucaran** [comment]
A few comments.

The question: how to install the latest fish on Linux? But what I meant was, how to get the latest fish build, so the point is to provide compilation instructions that just work. 

I liked the idea at first, but I am not so sure anymore :)

**Comment by bucaran** [comment]
@herrbischoff This is too difficult to maintain.

Can you redo this with the following changes:

Remove question 2 and 3, also please update the other READMEs.

**Comment by bucaran** [comment]
@herrbischoff I'll take care of it. Nevermind :)

**Comment by herrbischoff** [comment]
Alright!

**Pull request closed without merging**